### PR TITLE
Bot bot!

### PR DESCRIPTION
**Pull request by errietta**

### CODE REVIEW

**Comment by errietta** [comment]
@thisandagain R?

**Comment by thisandagain** [comment]
\o/

/cc @k88hudson @gvn

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# webmaker-app-bot
+Webmaker announce bot.
```
Comment (thisandagain) [inline README.md]: We've been trying to keep all issues in github.com/mozilla/webmaker-app. Perhaps we could add a line here:

```
#### Issues
Please file issues for the announce bot [here](https://github.com/mozilla/webmaker-app/issues).
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "webmaker-irc-notifier",
```
Comment (thisandagain) [inline package.json]: Let's call it `webmaker-app-bot`

**Comment by davidascher** [comment]
Drive-by: I broke it so it's not live today (need to figure out why), but
worth knowing about https://github.com/MozillaFoundation/signal as well --
a Hubot install to find out about new github issues, supposed to pipe up in
#webmaker-signal.

**Comment by errietta** [comment]
@davidascher that is cool :+1:

**Comment by errietta** [comment]
@thisandagain  Re-R?

**Comment by errietta** [comment]
@thisandagain @k88hudson i added the ability to have separate config files now :smile: 
is default.conf ok?

File: app.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+if (environment === 'STAGING') {
+  console.log('loading staging config');
+  habitat.load('config/staging.env');
+} else if (environment === 'MOFODEV') {
+  console.log('loading mofodev config');
+  habitat.load('config/mofodev.env');
+}
+
+habitat.load('config/defaults.env');
```
Comment (k88hudson) [inline app.js]: add this before line 11

Comment (k88hudson) [inline app.js]: .env should be loaded first

**Comment by k88hudson** [comment]
@errietta one more small change and this good to go

**Comment by errietta** [comment]
@k88hudson .load doesn't override, so if i put defaults.env first you'll end up using defaults always. You're supposed to have it last. Unless that's not what you meant?

**Comment by k88hudson** [comment]
R+

**Comment by thisandagain** [comment]
:shipit:

**Pull request merged**